kotlin/show-device-location-using-indoor-positioning/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.esri.arcgisruntime.sample.showdevicelocationusingindoorpositioning">
+
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
+    <uses-permission android:name="android.permission.BLUETOOTH"/>
+    <uses-permission android:name="android.permission.BLUETOOTH_ADMIN"/>
```

Comment:
`BLUETOOTH_ADMIN` enables the device to initiate device discovery or manipulate Bluetooth settings. This would be needed to discover and connect to multiple beacons in an IPS environment.